app/models/concerns/twitter_user/api.rb

```diff
@@ -0,0 +1,262 @@
+require 'active_support/concern'
+
+module Concerns::TwitterUser::Api
+  extend ActiveSupport::Concern
+
+  included do
+    def one_sided_following
+      ExTwitter::Client.new.one_sided_following(self)
+    end
+
+    def one_sided_followers
+      ExTwitter::Client.new.one_sided_followers(self)
+    end
+
+    def mutual_friends
+      ExTwitter::Client.new.mutual_friends(self)
+    end
+
+    def common_friends(other)
+      return [] if other.blank?
+      ExTwitter::Client.new.common_friends(self, other)
+    end
+
+    def common_followers(other)
+      return [] if other.blank?
+      ExTwitter::Client.new.common_followers(self, other)
+    end
+
+    def removing
```

Comment:
__[rubocop]__
* Assignment Branch Condition size for removing is too high. [16.03/15] [:ref](http://c2.com/cgi/wiki?AbcMetric)